community/repo_guidelines.md

```diff
@@ -6,12 +6,12 @@ The ONNX GitHub organization contains a number of repositories. Every repository
 
 ## Rules for all repos
 
-* Must be owned and managed by one of the ONNX SIGs
+* Must be owned and managed by one of the ONNX SIGs or the Steering Committee
 * Must be actively maintained
 * Must adopt the ONNX Code of Conduct
-* Must adopt the standard ONNX license(s)
+* Must adopt the standard ONNX license(s) [All code projects use the Apache 2.0 license. Documentation repositories must use the Creative Commons License version 4.0.]
 * Must adopt the ONNX DCO bot
-* Must adopt all ONNX automation (like LGTM)
+* Must adopt all ONNX automation (like static code analysis)
```

Comment:
I find this  bit strong, "Must adopt all ONNX automation", not even sure what it means. Can we change it to "Should strive to adopt all ONNX automation" or maybe "Must adopt all relevant ONNX automation".

Reply:
We can make it specific: Must enable automated static code analysis.
If there are other requirements to meet LFAI or OSS guidelines, we can list them here explicitly